### PR TITLE
MCKIN-24785. Platform issues - Instructor tool xblock strings for translations

### PR DESCRIPTION
**Pull request by wasifarbisoft**
https://edx-wiki.atlassian.net/browse/MCKIN-24785

### CODE REVIEW

**Comment by wasifarbisoft** [comment]
@xitij2000 Please review this Pull Request

File: problem_builder/mixins.py [hunk 1]
```diff
@@ -15,6 +15,20 @@
 
 loader = ResourceLoader(__name__)
 
+PB_LANGUAGE_JS_DIRECTORY_MAP = {
```
Comment (aayushagra) [inline problem_builder/mixins.py:18]: @xitij2000 @wasifarbisoft Shouldn't this data be loaded from a configuration file instead of being hardcoded?

**Review by aayushagra — CHANGES_REQUESTED** [review]
@wasifarbisoft Why are there pointless merge commits merging PR #1  and #4 ? Can you clean up the commit history please

**Comment by wasifarbisoft** [comment]
@aayushagra ,
I was unable to remove the pointless commit form this PR maybe they were added due to the stale branch I had for Problem

I am closing this Pull Request and created a new branch with clean commits
New Branch is https://github.com/open-craft/problem-builder/pull/307

**Pull request closed without merging**